.github/workflows/mvn-verify-check.yml

```diff
@@ -130,14 +137,31 @@ jobs:
       - name: Build JDK11
         run: >
           mvn -Dmaven.wagon.http.retryHandler.count=3 -B verify
-          -P 'individual,pre-merge'
+          -P 'individual,pre-merge,jdk11-build'
+          -Dbuildver=${{ matrix.spark-version }}
+          -DskipTests
+          -Dskip
+          -Dmaven.javadoc.skip
+
+  verify-modules-with-jdk17:
```

Comment:
the proliferation of this kind of copy and paste continues because we are not adding  jdk versions as a dimension to the build matrix

Reply:
sure I will add a TODO here. If we are going to add more jdks, then we can refactor these to matrix later.
currently I would like to keep it as it is for now